modules/features/falcon_feature_commerce_direct_debit_gateway/falcon_feature_commerce_direct_debit_gateway.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Falcon Commerce Direct Debit Gateway feature'
```

Comment:
Could you please explain the value of this feature? From what I see it just enables one contrib module https://www.drupal.org/sandbox/duozersk/3005285 and that's it. I'd enable it directly in the client project just like we did for Realex (tests can live in client project too). What do you think?

Reply:
It adds the payment gateways - test and live one, for the direct debits.